### PR TITLE
Bunch of improvements to the update scripts

### PR DESCRIPTION
**Pull request by indradhanush**
Each commit is an independent update but too small to create a different PR on it's own. I hope looking at the commits at a time for review would make it easier.

### CODE REVIEW

File: scripts/updates/update-kubelet [hunk 1]
```diff
@@ -22,7 +22,7 @@ upgrade_kubelet() {
 }
 
 wait_for_node_ready() {
-  local status
+  local state
```
Comment (surajssd) [inline scripts/updates/update-kubelet]: this slipped off of us last time ;-)

**Comment by surajssd** [comment]
nit: https://github.com/kinvolk/kubernetes-the-hab-way/pull/22/commits/1df61bfc4ff115cef502982c0e12ad057859c0ba commit has spell mistake of `components`

**Comment by indradhanush** [comment]
Fixed. It's now known as e6d6b5a.

File: scripts/updates/update-kubelet (hunk 1 above)
Comment (indradhanush) [reply scripts/updates/update-kubelet]: :sweat_smile:

**Comment by indradhanush** [comment]
Merged on the commandline.

**Pull request merged**